master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java

```diff
@@ -74,6 +74,8 @@ public abstract class AbstractMetaManager implements IMetadataHandler {
   public long estimatedPartitionSize;
   public final LongAdder partitionTotalWritten = new LongAdder();
   public final LongAdder partitionTotalFileCount = new LongAdder();
+  public final AtomicLong partitionWritten = new AtomicLong();
```

Comment:
This value looks like they didn't need to sync across all master nodes so they should be in AbstractMetaManager. Only variables that need to sync across all master nodes shall be placed in this class.